templates/chat/roll/channel-card.html

```diff
@@ -6,7 +6,9 @@
     {{localize "Overchannelling"}}: <span class="overchannelling-count"><b>{{test.result.overchannelling}}</b></span> 
     </div>
   {{/if}}
+  {{#if user.isGM}}
     <div class="card-content">
       {{localize "SHEET.CN"}}: <span class="spellcn-count"><b>{{test.result.CN}}</b></span> 
     </div>
+  {{/if}}
 </div>
```

Comment:
Now that I'm at my computer: 

Notice the `hide-option` class
https://github.com/moo-man/WFRP4e-FoundryVTT/blob/a5d52c49cc54a215b61e306259de6345d63bad0d/templates/chat/roll/test-card.html#L23

Removed in the `renderChatMessage` hook
https://github.com/moo-man/WFRP4e-FoundryVTT/blob/a5d52c49cc54a215b61e306259de6345d63bad0d/modules/hooks/chat.js#L285

Note that the code is *very* old and I haven't looked at it in a long time. I would presume it would still work by just adding `.hide-option` to the element.

Reply:
Ok, done this way, thanks.

Reply:
Ok, i noticed your second comment a bit later, but i think my dedicated class and dedicated removal will be better, as i dont want to interfere and depend on the roll-test setting